platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -16,46 +16,104 @@
 
 package com.swirlds.common.system.address;
 
+import static com.swirlds.common.system.address.Address.ipString;
+
+import com.swirlds.common.formatting.TextTable;
 import com.swirlds.common.system.NodeId;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 import java.text.ParseException;
 import java.util.Objects;
-import java.util.function.Function;
 
 /**
  * A utility class for AddressBook functionality.
+ * <p>
+ * Format for the config.txt address book: keywork "address" followed by a comma, followed by the node ID, followed by a
+ * comma, followed by the nickname, followed by a comma, followed by the self name, followed by a comma, followed by the
+ * weight, followed by a comma, followed by the internal IP address, followed by a comma, followed by the internal port,
+ * followed by a comma, followed by the external IP address, followed by a comma, followed by the external port,
+ * followed by a comma, followed by the memo.  The memo is optional.  Each address line is terminated by a newline ("\n"
+ * or "\r\n") or end of file.
```

Comment:
Perhaps you could rewrite this in the following form:

```
Each line in the config.txt address book contains the following elements, separated by commas. Each address line is terminated by a newline ("\n" or "\r\n") or end of file.

<ul>
<li>the keyword "address"</li>
...
<li>a memo (optional)</li>
</ul>
```

Reply:
will do